map/map-test.js

```diff
@@ -197,7 +197,7 @@ QUnit.test("set multiple props", function(){
 	QUnit.deepEqual(map.get(), {foo: {bar: "VALUE"}, a: 2}, "works nested");
 });
 
-QUnit.test("serialize responds to added props", function(){
+QUnit.test("serialize responds to added props", function(assert) {
```

Comment:
Why is adding `assert` here important if `QUnit.deepEqual` is used?

Reply:
@justinbmeyer `assert` is important because of its assertion methods (`assert.equal`, `assert.ok` ... etc) and also `expect` function is a method in the `assert` object